arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/IotHubDescription"
+						}
+					},
+					"202": {
+						"description": "Accepted",
+						"schema": {
+							"$ref": "#/definitions/IotHubDescription"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/eventHubEndpoints/{eventHubEndpointName}/ConsumerGroups/{name}": {
+			"delete": {
+				"tags": ["DELETE"],
+				"summary": "Delete EventHub Consumer Group for an IotHub",
+				"description": "Delete EventHub Consumer Group for an IotHub",
+				"operationId": "IotHubResource_DeleteEventHubConsumerGroup",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "eventHubEndpointName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "name",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/Object"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/providers/Microsoft.Devices/operations": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all supported Resource Provider Operations",
+				"description": "Get all supported Resource Provider Operations",
+				"operationId": "IotHubResource_GetResourceProviderOperations",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResponseWithContinuation[IEnumerable[ResourceProviderOperationInfo]]"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Devices/IotHubs": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all IotHubs in a subscription",
+				"description": "Get all IotHubs in a subscription",
+				"operationId": "IotHubResource_GetIotHubsForSubscription",
```

Comment:
This is the expected format IotHubResource_ListBySubscription
https://github.com/Azure/azure-rest-api-specs/issues/244
@selvasingh - Catching this right before the spec is being born ;).